src/components/AddBlockButton/AddBlockButton.tsx

```diff
@@ -39,12 +67,17 @@ export const AddBlockButton = ({
             className={merge([
```

Comment:
Here I would simplify using the mapping. We still need to add a default so I would add `tw-rotate-0` in the first line of styling. 

```
className={merge([
                'tw-group tw-leading-none tw-rounded-sm tw-outline-none tw-rotate-0',
                isFocusVisible && FOCUS_STYLE,
                orientation && !direction && orientationMap[orientation],
                direction && directionMap[direction],
            ])}
```

Reply:
if you give a default value for direction, this line `orientation && !direction && orientationMap[orientation],` may become useless 🤔